### PR TITLE
Add hec_index_checker

### PR DESCRIPTION
**Pull request by akinnane**

### CODE REVIEW

**Review by danjoneslf — COMMENTED** [review]
Can you add something to the README to explain what this repo is for and what's in it? 

If cst.py is a click cli should it be called cli.py? 

I liked tox but we moved away from it - why has this gone back to tox?

**Comment by akinnane** [comment]
> Can you add something to the README to explain what this repo is for and what's in it?

Yes, briefly though - this is going to become our single repo for all of our python code.

>If cst.py is a click cli should it be called cli.py?

I went back and forth a few times between the two names. The runnable artefact from this is called `cst`(maybe needs a better name) but I eventually settled on calling this file the same as the script name. I found it confusing when the command was `cst` but the file is `cli.py`. I'm not 100% convinced about either thought so we can discuss this offline. 

>I liked tox but we moved away from it - why has this gone back to tox?

I like tox too. Specifically I like that it lets us test against multiple python versions and as this library will be included in many projects we will need to run the tests against all the python versions we use.

**Comment by danjoneslf** [comment]
> The runnable artefact from this is called `cst`

What do you mean - runnable artefact? I'm assuming you run `python cst.py check-hec-token --blah`. Are you saying we'd package that up into an executable somehow so you run `cst check-hec-token --blah`?

**Comment by akinnane** [comment]
Ideally, yes. That way we can have a single command to install that would give us access to useful entrypoints and functions in the code. Hopefully that would make developing, CI, and debugging much easier to work with.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Test files live along side the python files. They have the same name
+with a suffix of `_test`.  For example if your file is
+`cybersecuritytools/foo/bar.py` then the tests will be in
+`cybersecuritytools/foo/bar_test.py`.
+
+# Committing
+* Add documentation for any changes you've made.
+* Ensure there are tests covering your changes
+* Update the (detect-secrets)[https://github.com/Yelp/detect-secrets] file (`.secrets.baseline`)[.secrets.baseline]
+* Run `tox`, all tests, formatting, and linting must pass.
```
Comment (danjoneslf) [inline README.md:63]: Why doesn't `make test` run `tox`?

**Review by danjoneslf — APPROVED** [review]
I like it. It feels like we need a bit of directory structure separate the things that are helper modules to be used across our python projects to make it clear that's what they are.

**Comment by akinnane** [comment]
Yes, we will need some sort of structure. As I move more code into this repo I'm hoping that will develop in the right way. If not we can reorganise.

**Pull request merged**